src/SFA.DAS.Commitments.Application/Commands/RejectTransferRequest/RejectTransferRequestCommandHandler.cs

```diff
@@ -62,6 +62,19 @@ public class RejectTransferRequestCommandHandler: AsyncRequestHandler<RejectTran
 
             _historyService.TrackUpdate(commitment, CommitmentChangeType.TransferSenderRejection.ToString(), commitment.Id, null, CallerType.TransferSender, command.UserEmail, commitment.ProviderId, command.TransferSenderId, command.UserName);
 
+            if (command.TransferRequestId > 0)
+            {
+                await _commitmentRepository.SetTransferRequestApproval(command.TransferRequestId, command.CommitmentId,
+                    TransferApprovalStatus.TransferRejected, command.UserEmail, command.UserName);
+            }
+            else
+            {
+                // TODO Remove This route when old Approval route decomes obslete 
+                await _commitmentRepository.SetTransferApproval(command.CommitmentId, TransferApprovalStatus.TransferApproved,
```

Comment:
I think this should be .TransferRejected...

Reply:
Good spot